mmpose/datasets/datasets/top_down/topdown_posetrack18_video_dataset.py

```diff
@@ -215,9 +214,6 @@ def _load_coco_keypoint_annotation_kernel(self, img_id):
                 if osp.exists(sup_image_file):
```

Comment:
Please add some comments here to describe the relationship between `sup_image_file` and `cur_image_file` and when they will be used respectively?

Reply:
The original purpose of these lines is to use `osp.exists` to check whether the `sup_image_file` exists locally. The frame indices have been checked above, which will make sure that they do not exceed the bounding of frame indices. But when the training data is on Ceph, every `sup_image_file` does not exist locally,  which will lead to a large amount of warnings. 

So, I  suggest that we can leave the file existence check in the pipeline like `LoadImageFromFile` instead of inside this dataset. And change these lines into:
```
image_files.append(sup_image_file)
```
Do you think this is proper?

Reply:
Yes, we can check the image file's existence in the pipeline.